packages/framework/src/Resources/config/services.yml

```diff
@@ -13,6 +13,15 @@ services:
     autoconfigure: true
     public: false
 
+  Shopsys\FrameworkBundle\:
+      resource: '../../{Component,Model,DataFixtures}/**/{*Resolver.php,*Factory.php,*Service.php,*Facade.php,*Repository.php,*Listener.php,*Loader.php}'
+
+  Shopsys\FrameworkBundle\Component\:
+    resource: '../../Component/**/{*Provider.php,*Transformer.php,*Finder.php,*Parser.php,*DataCreator.php}'
+
+  Shopsys\FrameworkBundle\Model\:
+    resource: '../../Model/**/{*Scheduler.php,*Generator.php,*Calculation*.php,*Recalculator.php,*Filter.php,*Mail.php,*Setting.php,*Config.php,*DataMapper.php}'
```

Comment:
`../../Model/**/*Calculation*.php` will match for example `\Shopsys\FrameworkBundle\Model\Product\Exception\InvalidPriceCalculationTypeException`, which should not be defined as a service.

I suggest auto-discovering only by a postfix, i.e. using `../../Model/**/*Calculation.php`. This would not match `Model/Product/Pricing/ProductPriceCalculationForUser`, this class should be defined explicitly (as it is an exception from the rule).